src/Tooling/Bit.Tooling.Templates/TodoTemplate/App/TodoTemplate.App.csproj

```diff
@@ -12,7 +12,7 @@
         <ApplicationTitle>TodoTemplate</ApplicationTitle>
 
         <!-- App Identifier -->
-        <ApplicationId>com.companyname.TodoTemplate</ApplicationId>
+        <ApplicationId>com.bitplatform.TodoTemplate</ApplicationId>
```

Comment:
again, this change is not related to this issue.
plz remove it and submit a new issue.